scripts/basic-workflow-tests.sh

```diff
@@ -296,8 +301,55 @@ test_train() {
 
         ilab model train "${TRAIN_ARGS[@]}"
     fi
+}
+
+test_phased_train() {
+    task Train the model with LAB multi-phase strategy.
+
+    # linter wants DATA_PATH and over variables declared then assigned separately.
+    # 'Declare and assign separately to avoid masking return values' <-- error.
+    local DATA_PATH
+    DATA_PATH=$(find "${DATA_HOME}"/instructlab/datasets -name 'messages_*' | head -n 1)
```

Comment:
This `messages_*` file is a legacy SDG output and not the latest one. SDG output now has two dataset outputs, one for knowledge and one for skills called `knowledge_train_msgs_*.jsonl` and `skills_train_msgs_*.jsonl`.